test/runAll.unmocked.spec.js

```diff
@@ -843,6 +843,31 @@ describe('runAll', () => {
     expect(await execGit(['log', '-1', '--pretty=%B'], { cwd: workTreeDir })).toMatch('test')
     expect(await readFile('test.js', workTreeDir)).toEqual(testJsFilePretty)
   })
+  ;['on', 'off'].forEach(quotePath => {
```

Comment:
Here you could use the `test.each` syntax:

```js
test.each([
   ['on'],
  ['off']
])('should handle files with non-ascii characters when core.quotepath is %i', quotePath => {
    ...
```

Reply:
Oh wow I did not know about this syntax despite using Jest for years. Thanks, will change now 😉

Reply:
✅ done